packages/nimble/R/genCpp_sizeProcessing.R

```diff
@@ -368,7 +368,7 @@ sizeRecyclingRuleRfunction <- function(code, symTab, typeEnv) {
     code$sizeExprs <- newSizeExprs
     code$type <- 'double' ## will need to look up from a list
     code$nDim <- 1
-    code$toEigenize <- TRUE
```

Comment:
Gosh, was this a bug due to a type error? If we do separate out lifting-for-eigenization (as we discussed yesterday), it's probably worth checking the type in the one place where the field will be read.